publication-commons/src/main/java/no/unit/nva/publication/model/business/TicketStatus.java

```diff
@@ -13,6 +15,7 @@
 import nva.commons.core.SingletonCollector;
 
 public enum TicketStatus {
+    NEW(NEW_STATUS, "NEW"),
```

Comment:
The TicketStatus enum is used in the TicketEntry-model which is saved in the database, so we risk that Tickets are created and saved in the database with status "New" (which we don't want).
Suggestion: create a new TicketStatus Enum which is used is expanded tickets, the new Enum can include status New, Pending, Completed and Closed.

Reply:
Or make status a String in expanded tickets. @ketilaa FYI